tests/integration/conftest.py

```diff
@@ -805,4 +808,4 @@ async def ingress_application_related_fixture(application: Application, external
         idle_period=30,
         raise_on_error=False,
     )
-    return traefik
+    return application
```

Comment:
Although this can makes sense, checking for example other fixtures like `prometheus_related`, `grafana_related` return the prometheus or the grafana app, so it is not totally consistent. Maybe returning here traefik and using the application in the test?

Reply:
Changed